src/jwt.ts

```diff
@@ -147,7 +147,7 @@ export const verify = (options: JWTVerifyOptions) => {
   }
 
   /* c8 ignore next 5 */
-  if (!isNumber(decoded.claims.exp)) {
+  if (decoded.claims.exp == null || !isNumber(decoded.claims.exp)) {
```

Comment:
This looks like it adds a check, but this is also checked as part of `isNumber`, but it's not clear it does. Also TS strict likes it when we check for `null` or `undefined` explicitly.